### ISSUE TITLE
“Page” and “Script” tags in sidebar contrast should be AA compliant

### ISSUE THREAD

**Issue by phacks** [theodo/falco#140]
Thanks @AlbericTrancart for the report.

From https://a11yproject.com/checklist/#section-color-contrast:

> Level AA compliance requires a contrast ratio of 4.5:1.

![Current contrast is around 2](https://user-images.githubusercontent.com/2587348/77773678-3e19f200-704a-11ea-92a3-39b3729e0e5f.png)

Current contrast on tags is around 2:1, should be at or above 4.5:1
